docs/shared/getting-started/intro.md

```diff
@@ -1,8 +1,18 @@
 # Intro to Nx
 
-Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality. Find out more about [why you should use Nx](/getting-started/why-nx).
+Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality.
 
-If instead you want to jump right into it, run the following command. It will guide you through the setup:
+## Key Features
+
+- **Efficient Task Execution**: Nx runs tasks in parallel, keeping potential dependencies among them in mind
+- **Local & Remote Caching**: With local and remote caching, Nx prevents unnecessary re-runs of tasks, saving you valuable dev time.
+- **Smooth Code Migrations**: Leveraging Nx plugins means gaining automatic code generation and migration tools - useful for keeping NPM packages up-to-date and minimizing breaking changes.
+- **Editor Integration**: Nx ships with Nx Console, an extension for VSCode & IntelliJ (and an LSP for you Vim nerds out there)
+- **Make it Your Own**: The [Nx Devkit](/plugins/intro/getting-started) lets you extend Nx's capabilities with your own automation, or you can even build your CLI on top of it.
+
+Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [core features](/core-features) section.
+
+## Try Yourself!
```

Comment:
```suggestion
## Try it yourself!
```

Reply:
I believe you can also say "Try Yourself"... Not sure whether if you add "it" you then need to say "Try it by yourself"

Reply:
Yep, this is what I intended. That's why I went for "Try Yourself" :)

<img width="687" alt="image" src="https://github.com/nrwl/nx/assets/542458/4826a8f7-6feb-4e03-8855-3b0fc319ff0d">